CONTRIBUTING.md

```diff
@@ -0,0 +1,229 @@
+
+2. **Clone Your Fork**
+
+   ```bash
+   git clone git@github.com:availproject/availproject.github.io.git
+   ```
+
+3. **Navigate to the Repository**
+
+   ```bash
+   cd availproject.github.io
+   ```
+
+4. **Add Upstream Remote**
+
+   > Refer to [GitHub Docs: Configuring a remote for a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork).
+
+   ```bash
+   git remote add upstream https://github.com/availproject/avail-docs
+   ```
+
+5. **Sync Your Fork**
+
+   > See [GitHub Docs: Syncing a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork).
+
+   ```bash
+   git checkout master
+   git fetch upstream
+   git merge upstream/master
+   ```
+
+6. **Install Dependencies**
+
+   ```bash
+   yarn install
+   ```
+
+   The site is built using Docusaurus. You may need to install Docusaurus before running the documentation locally.
+
+   ```bash
+   yarn add docusaurus
+   ```
+
+   Alternatively, you can upgrade Docusaurus.
+
+   ```bash
+   yarn upgrade @docusaurus/core@latest @docusaurus/preset-classic@latest
+   ```
+
+7. **Run the Docs Locally**
+
+   ```bash
+   yarn start
+   ```
+
+## License
+
+The Avail Project Developer Documentation is licensed under the [MIT License](./LICENSE) free software license.### How to Contribute Changes via the Avail Documentation Website
+
+Contributing to the Avail Documentation is simple. You'll need a GitHub account and a basic understanding of Markdown syntax to get started.
+
+1. **Locate the Page**: Visit the [Avail Documentation page](https://docs.availproject.org/) you wish to edit.
+2. **Navigate to the Bottom**: Scroll to the bottom of the page.
+3. **Edit Link**: Click on the **Edit this page** link. This will redirect you to the corresponding Markdown file on GitHub.
+4. **Edit Mode**: Once on GitHub, click the pencil icon located in the upper-right corner to enter edit mode.
+5. **Make Edits**: Modify the Markdown file as needed.
+6. **Initiate Pull Request**: Scroll to the bottom and click on **Create pull request**.
+7. **Title Your PR**: Give your pull request a descriptive title. For example, if you're editing the "Getting Started" page, you could title it _Update /docs/getting-started.md_.
+8. **Describe Changes**: In the pull request description, specify the issue your changes resolve.
+   > See [GitHub Docs on Linking a Pull Request to an Issue](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for guidance.
+9. **Additional Information**: Provide a concise summary of the changes you've made. Include screenshots or references if applicable.
+10. **Submit**: Click **Propose changes** to finalize your pull request. This will create a new branch in your fork.
+
+### How to Locally Run the Avail Documentation
```

Comment:
this section is repeated https://github.com/availproject/availproject.github.io/pull/194/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R59

Reply:
Damn. Missed it. My bad. The whole thing after the LICENSE heading was repeated.